main.go

```diff
@@ -19,47 +19,60 @@ import (
 	"github.com/urfave/cli/v2/altsrc"
 )
 
-const redColor = "\u001B[31m"
-const greenColor = "\u001B[32m"
-const blueColor = "\u001B[36m"
-const endColor = "\u001B[0m"
```

Comment:
I would not remove these constants. Readability of following code will be only worse.